### PR TITLE
Used space encryption and CentOS 7.6 support[DualPass]

### PR DESCRIPTION
**Pull request by vimish**

### CODE REVIEW

File: VMEncryption/main/oscrypto/rhel_72/encryptstates/EncryptBlockDeviceState.py [hunk 1]
```diff
@@ -25,6 +25,7 @@
 from inspect import ismethod
 from time import sleep
 from OSEncryptionState import *
+from distutils.version import LooseVersion
```
Comment (vermashi) [inline VMEncryption/main/oscrypto/rhel_72/encryptstates/EncryptBlockDeviceState.py]: Is distutils something that is pre-installed with Python?

Comment (vimish) [reply VMEncryption/main/oscrypto/rhel_72/encryptstates/EncryptBlockDeviceState.py]: Yes...Also, I am using the same to check for kernel version comparison
https://github.com/Azure/azure-linux-extensions/blob/ade-singlepass-master/VMEncryption/main/check_util.py#L270

Comment (vermashi) [reply VMEncryption/main/oscrypto/rhel_72/encryptstates/EncryptBlockDeviceState.py]: Nice! Maybe we can use this to simplify all the version checking elsewhere in the code when we get the chance.

**Comment by vermashi** [comment]
Looks good.
Test on Centos 7.6 and Centos 7.6 LVM and then merge it.

**Pull request merged**